src/Components/RightSidebar.elm

```diff
@@ -53,7 +63,7 @@ view { showRightSidebarMenu, showRightSidebarMenuMsg, clearPlaylistMsg, refreshP
                                     Local ->
                                         []
                                 )
-                                (text " Kodi")
+                                (text "  Kodi")
```

Comment:
Is this an oversight?

Reply:
Yes, it was an oversight. Upon realizing that removing the empty space in the string changes the spacing of tabs, I have fixed it in the code.